theano/tensor/tests/test_opt.py

```diff
@@ -3558,8 +3551,8 @@ def test0(self):
         if isinstance(mode_opt, compile.DebugMode):
             self.assertRaises(ValueError, f)
 
-        # No need to check_stack_trace
-        #self.assertTrue(check_stack_trace(f, ops_to_check='all'))
+        # No need to check_stack_trace as the graph only contains
+        # DeepCopyOp which has empty trace
```

Comment:
The issue is not with DeepCopyOp, you can just say that the optimization `local_useless_alloc` only removes nodes.